code/game/objects/items/bodybag.dm

```diff
@@ -51,13 +52,20 @@
 			name = "body bag - "
 			name += t
 			overlays += "bodybag_label"

```

Comment:
Sorry I forgot to mention this before, but this can be replaced with a call of update_icon now after else